sdk/include/crt/sal.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * PROJECT:     ReactOS PSDK
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Standard Annotation Language (SAL) definitions
+ * COPYRIGHT:   2021 - Jérôme Gardou
+ */
+
+#pragma once
+
+/* Include MS version first */
+#include "msc/sal.h"
```

Comment:
Having moved the sal.h headers out of include/psdk, breaks our psdk compatibility with MS.

Reply:
Alright. Although I would have preferred to have this in a compiler-specific directory, but so be it.

Reply:
We can imagine a hybrid solution where the include/psdk also contains compiler-specific subdirectories, and the "master" sal.h is in include/psdk . Then it includes the compiler-specific sal headers that are in include/psdk/msc or gcc. 🤔

Reply:
well, for now there would only be ms_sal.h & gcc_sal.h so I think this can stay as it is. If more files need splitting, we can create a subdirectory, yes.